content/pt-br/contributor-ladder/_index.md

```diff
@@ -0,0 +1,99 @@
+
+### Aprovadores de localização
+
+O glossário possui também aprovadores de localização. Estes são aprovadores para um dos times de localização (times que traduzem o glossário). Aprovadores de localização apenas tem permissões para aprovar PRs para seu próprio time e também fazer o merge na branch de desenvolvimento dedicada ao seu time. Um aprovador de localização também pode se tornar um aprovador do glossário em inglês caso atenda os requisitos.
+
+### Se tornando um aprovador
+
+Os candidatos a aprovador devem ter um histórico comprovado de envio de PRs de alta qualidade e de ajudar outras pessoas a obter deixar seus PRs prontos para o merge. Se o fuso horário permitir, eles também devem participar regularmente das [reuniões do Grupo de Trabalho do Glossário](https://www.cncf.io/calendar/).
+
+Para se tornar um aprovador, comece manifestando interesse aos mantenedores existentes. Os mantenedores existentes solicitarão que você demonstre as qualificações acima, contribuindo com PRs, fazendo revisões e realizando outras tarefas sob sua orientação. Depois de algum tempo trabalhando juntos, os mantenedores decidirão se concederão a você o status de aprovador. Esta decisão será baseada em seu nível demonstrado de proficiência e capacidade de resposta.
+
+## 3. Mantenedores
+
+Mantenedores são aprovadores que também podem fazer o merge de PRs. Qualquer pessoa pode se tornar um mantenedor do glossário (veja "Se tornando um mantenedor"). Existem algumas responsabilidades esperadas dos mantenedores, incluindo:
+
+- Ser um aprovador ativo e responsável,
+- Ajudar a manter o repositório, incluindo configurações do site, permissões, templates de issues, workflows do Github e outras coisas,
+- Acompanhar o slack do glossário e ajudar sempre que possível,
+- Participar regularmente dos [encontros do Grupo de Trabalho do Glossário](https://www.cncf.io/calendar/)(sempre que for possível).
+
+Se um mantenedor não estiver mais interessado ou não puder realizar as atividades acima, ele deve ser movido para o status de emérito.
+
+### Se tornando um mantenedor
+
+Os mantenedores devem ter um histórico comprovado de aprovação de sucesso e envio de PRs de alta qualidade. Se o fuso horário permitir, eles também devem participar regularmente das reuniões do Grupo de Trabalho do Glossário.
+
+Para se tornar um mantenedor, comece expressando interesse aos mantenedores existentes. Os mantenedores existentes solicitarão que você demonstre as qualificações acima, contribuindo com PRs, fazendo revisões e realizando outras tarefas sob sua orientação. Depois de algum tempo trabalhando juntos, os mantenedores decidirão se concederão o status de mantenedor. Esta decisão será baseada no nível demonstrado de proficiência e capacidade de resposta.
+
+## 4. Community Managers
```

Comment:
Not sure if here would be nicer `Mantenedores de comunidade`?

Reply:
Hmm, it's related to the GitHub repository, I'm not sure If we can use `mantenedores de comunidade`  what you think @brunoguidone?

Reply:
Sounds good! ^